go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/rs/cors v1.10.1
 	github.com/sassoftware/relic v7.2.1+incompatible
 	github.com/secure-systems-lab/go-securesystemslib v0.8.0
-	github.com/sigstore/sigstore v1.8.1
+	github.com/sigstore/sigstore v1.8.2-0.20240215213640-922069a3bc39
```

Comment:
can you also bump to v1.8.2, we just cut a release

Reply:
Sure, doing now!